llvm/test/Instrumentation/MemorySanitizer/SPIRV/instrument_global_address_space.ll

```diff
@@ -0,0 +1,140 @@
+; RUN: opt < %s -passes=msan -msan-instrumentation-with-call-threshold=0 -msan-eager-checks=1 -S | FileCheck %s
+
+; ModuleID = 'check_call.cpp'
+source_filename = "check_call.cpp"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64-G1"
+target triple = "spir64-unknown-unknown"
+
+$_ZTSZZ4mainENKUlRN4sycl3_V17handlerEE_clES2_E8MyKernel = comdat any
+
+; CHECK: @__MsanKernelMetadata
+
+; Function Attrs: mustprogress norecurse nounwind sanitize_memory uwtable
+define weak_odr dso_local spir_kernel void @_ZTSZZ4mainENKUlRN4sycl3_V17handlerEE_clES2_E8MyKernel(ptr addrspace(1) noundef align 4 %_arg_array) local_unnamed_addr #0 comdat !srcloc !85 !kernel_arg_buffer_location !86 !sycl_fixed_targets !87 {
+; CHECK-LABEL: @_ZTSZZ4mainENKUlRN4sycl3_V17handlerEE_clES2_E8MyKernel
+entry:
+; CHECK-NOT: @__msan_param_tls
```

Comment:
Just double-checking that you are aware that `CHECK-NOT` is scoped and this check allows `__msan_param_tls` appear after `__msan_get_shadow`

Reply:
Thanks for your reminder. This CHECK-NOT is on purpose.